modules/calib3d/src/usac/gamma_values.hpp

```diff
@@ -1,237 +1,120 @@
-0.00638,0.00637,0.00636,0.00635,0.00634,0.00633,0.00632,0.00631,0.0063,0.00629,0.00628,0.00627,0.00626,0.00625,0.00624,0.00623,0.00622,0.00621,0.00619,0.00618,0.00617,0.00616,0.00615,0.00614,0.00613,0.00612,0.00611,0.0061,0.00609,0.00608,0.00607,0.00606,0.00605,0.00604,0.00603,0.00602,0.00601,0.006,0.00599,0.00598,0.00597,0.00596,0.00595,0.00594,0.00593,0.00592,0.00591,0.0059,0.0059,0.00589,0.00588,0.00587,0.00586,0.00585,0.00584,0.00583,0.00582,0.00581,0.0058,0.00579,0.00578,0.00577,0.00576,0.00575,0.00574,0.00573,0.00572,0.00571,0.0057,0.00569,0.00568,0.00568,0.00567,0.00566,0.00565,0.00564,0.00563,0.00562,0.00561,0.0056,0.00559,0.00558,0.00557,0.00556,0.00555,0.00555,0.00554,0.00553,0.00552,0.00551,0.0055,0.00549,0.00548,0.00547,0.00546,0.00545,0.00544,0.00544,0.00543,0.00542,
-0.00541,0.0054,0.00539,0.00538,0.00537,0.00536,0.00536,0.00535,0.00534,0.00533,0.00532,0.00531,0.0053,0.00529,0.00528,0.00528,0.00527,0.00526,0.00525,0.00524,0.00523,0.00522,0.00522,0.00521,0.0052,0.00519,0.00518,0.00517,0.00516,0.00516,0.00515,0.00514,0.00513,0.00512,0.00511,0.0051,0.0051,0.00509,0.00508,0.00507,0.00506,0.00505,0.00505,0.00504,0.00503,0.00502,0.00501,0.005,0.005,0.00499,0.00498,0.00497,0.00496,0.00495,0.00495,0.00494,0.00493,0.00492,0.00491,0.0049,0.0049,0.00489,0.00488,0.00487,0.00486,0.00486,0.00485,0.00484,0.00483,0.00482,0.00482,0.00481,0.0048,0.00479,0.00478,0.00478,0.00477,0.00476,0.00475,0.00474,0.00474,0.00473,0.00472,0.00471,0.00471,0.0047,0.00469,0.00468,0.00467,0.00467,0.00466,0.00465,0.00464,0.00464,0.00463,0.00462,0.00461,0.0046,0.0046,0.00459};
+constexpr double stored_complete_gamma_values_n2[] = {1.77245,1.7045,1.67638,1.65484,1.63669,1.62073,1.60631,1.59307,1.58075,1.5692,1.55829,1.54792,1.53803,1.52855,1.51944,1.51066,1.50218,1.49397,1.48601,1.47827,1.47074,1.46341,1.45626,1.44927,1.44245,1.43577,1.42924,1.42283,1.41655,1.41039,1.40434,1.3984,1.39256,1.38682,1.38117,1.37561,1.37013,1.36474,1.35942,1.35418,1.34901,1.34392,1.33889,1.33392,1.32902,1.32418,1.3194,1.31467,1.31001,1.30539,1.30083,1.29631,1.29185,1.28744,1.28307,1.27874,1.27446,1.27023,1.26603,1.26188,1.25777,1.25369,1.24965,1.24565,1.24169,1.23776,1.23387,1.23001,1.22618,1.22238,1.21862,1.21489,1.21119,1.20752,1.20387,1.20026,1.19668,1.19312,1.18959,1.18608,1.18261,1.17915,1.17573,1.17232,1.16895,1.16559,1.16226,1.15896,1.15567,1.15241,1.14917,1.14595,1.14275,1.13958,1.13642,1.13329,1.13017,1.12708,1.124,1.12094,
```

Comment:
Please don't explode binaries or OpenCV source code with such blobs.

Please use on-demand one-time initializer of such tables.

Reply:
fixed it

Reply:
> on-demand
> one-time initializer

possible implementation is a singleton: https://en.wikipedia.org/wiki/Singleton_pattern